crates/project-model/src/project_json.rs

```diff
@@ -320,8 +345,9 @@ struct CrateData {
     version: Option<semver::Version>,
     deps: Vec<Dep>,
     #[serde(default)]
-    #[serde(with = "cfg_")]
-    cfg: Vec<CfgAtom>,
+    cfg_extends: FxHashSet<String>,
```

Comment:
```suggestion
    cfg_groups: FxHashSet<String>,
```
I think that name fits better though either works I'd say